WordPress/gutenberg
# Documentation: Update docs tool to omit private, experimental, unstable APIs

Closes #15148

This pull request seeks to update the docs tool to omit selectors and actions which are either named as unstable or experimental, or which include a `@private` JSDoc tag.

- https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#experimental-and-unstable-apis
- http://usejsdoc.org/tags-private.html

**Implementation notes:**

There is certainly some overlap / potential for inconsistency between the documentation output by this tool, and that of `@wordpress/docgen`. These should probably be reconciled, but it seemed a larger effort than the immediate goal of excluding APIs not intended to be documented.

**Testing instructions:**

Verify there are no unstable, experimental, or private APIs documented.

Verify there are no local changes after running `npm run docs:build` in this branch.
